tools/db-versions.sh

```diff
@@ -1,15 +1,35 @@
-# Default cassandra version
-CASSANDRA_VERSION=${CASSANDRA_VERSION:-3.9}
+# Versions
```

Comment:
Doesn't this way of defining versions and then overriding them feel a bit redundant? Why's that better than the previous `VERSION=${VERSION:-XX.YY}`?

Reply:
It makes it easier to read (i.e. once version contains `-`, it kinda becomes a mess).

Reply:
I agree it's verbose. I merged the PR because we need tests fixed asap.